cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/TemporalFunctionsTest.scala

```diff
@@ -0,0 +1,1317 @@
+        | ** <<functions-duration-computing, Computing the _Duration_ between two temporal instants>>
+        |  *** <<functions-duration-between, duration.between()>>
+        |  *** <<functions-duration-inmonths, duration.inMonths()>>
+        |  *** <<functions-duration-indays, duration.inDays()>>
+        |  *** <<functions-duration-inseconds, duration.inSeconds()>>
+      """.stripMargin)
+    section("Temporal instant types (_Date_, _Time_, _LocalTime_, _DateTime_ and _LocalDateTime_)", "functions-temporal-instant-type") {
+      section("An overview of temporal instant type creation", "functions-temporal-create-overview") {
+        p(
+          """Each function bears the same name as the type, and construct the type they correspond to in one of four ways:
+            |
+            |* Capturing the current time
+            |* Composing the components of the type
+            |* Parsing a string representation of the temporal value
+            |* Selecting and composing components from another temporal value by
+            | ** either combining temporal values (such as combining a _Date_ with a _Time_ to create a _DateTime_), or
+            | ** selecting parts from a temporal value (such as selecting the _Date_ from a _DateTime_); the _extractors_ -- groups of components which can be selected -- are:
+            |  *** `date` -- contains all components for a _Date_ (conceptually _year_, _month_ and _day_).
+            |  *** `time` -- contains all components for a _Time_ (_hour_, _minute_, _second_, and sub-seconds; namely _millisecond_, _microsecond_ and _nanosecond_).
+            |  If the type being created and the type from which the time component is being selected both contain `timezone` (and a `timezone` is not explicitly specified) the `timezone` is also selected.
+            |  *** `datetime` -- selects all components, and is useful for overriding specific components.
+            |  Analogously to `time`, if the type being created and the type from which the time component is being selected both contain `timezone` (and a `timezone` is not explicitly specified) the `timezone` is also selected.
+            | ** In effect, this allows for the _conversion_ between different temporal types, and allowing for 'missing' components to be specified.""".stripMargin)
+        p(
+          """
+            |.Temporal instant type creation functions
+            |[options="header"]
+            ||===
+            || Function                   | Date | Time | LocalTime | DateTime | LocalDateTime
+            || Getting the current value  | <<functions-date-current, X>> | <<functions-time-current, X>> | <<functions-localtime-current, X>> | <<functions-datetime-current, X>> | <<functions-localdatetime-current, X>>
+            || Creating a calendar (Year-Month-Day) value | <<functions-date-calendar, X>> | | | <<functions-datetime-calendar, X>> | <<functions-localdatetime-calendar, X>>
```

Comment:
I would not call these `calander value` et al, but rather `calendar-based date value`.